docs/README.md

```diff
@@ -76,3 +76,11 @@ agent_list = [
     agents.PlayerAgent(agent_control="wasd"), # W,A,S,D to move, E to lay bomb
 ]
 ```
+
+## Submitting an Agent.
+
+In order to submit an agent, you need to create an account at [pommerman.com](https://pommerman.com). This will link pommerman to your Github account. 
+
+Once you have created an account login and navigate to your profile, [Pommerman profile](https://pommerman.com/me). Here you will see a page where you can fill in a form that includes an agent name, git url (ssh is preferred), and the path to your agent's Docker file.
+
+Next, you will need to add ssh keys to your account, so pommerman can access your agent's repo. The public key you need to add is provided in your Pommerman profile after registering an agent.
```

Comment:
"Next, you will need to add an ssh deploy key to your account so we can access your agent's repo. This is provided to you after registering the agent."